### ISSUE TITLE
Interest Invokers: avoid a11y traps

### ISSUE THREAD

**Issue by scottaohara** [openui/open-ui#1093]
In an offline discussion it was mentioned that interest invokers could potentially be used to invoke modal dialogs on hover or focus. While that is not mentioned in the explainer as a use case (my take away is that it's not a use case at all since the interestee elements are all listed as popovers), it should probably be mentioned as something _people should not do_ (e.g., by 'faking' a modal dialog by invoking a dialog popover and manually making everything inert).

In general, for the a11y section it should be noted that 'interest' does not necessarily indicate someone wants to immediately interact with whatever content is exposed / if they want to even interact with it at all.  So, it should be called out explicitly that it is not expected for a focus change to occur anytime an item of interest is displayed.  Doing so would commonly fail WCAG [3.2.1 - On Focus](https://www.w3.org/WAI/WCAG22/Understanding/on-focus.html).
